htdocs/takepos/ajax.php

```diff
@@ -46,6 +46,10 @@
     $object = new Categorie($db);
     $result=$object->fetch($category);
     $prods = $object->getObjectsInCateg("product");
+    function sort_by_ref($a, $b) {
```

Comment:
To prepare coming other features (like sorting product on a dedicated "position" field), it would be better to add a 'sort' parameter into method getObjectsInCateg() and include the sort into the method getObjectsInCateg.